webhooks-extension/config/latest/gcr-tekton-webhooks-extension.yaml

```diff
@@ -201,19 +201,19 @@ spec:
     image: maiwj/kubernetes-python-client:latest
     env:
       - name: PIPELINERUNS
-        value: ${inputs.params.pipelineruns}
+        value: $(inputs.params.pipelineruns)
```

Comment:
Folder name has changed from release to latest, https://github.com/tektoncd/experimental/blob/master/webhooks-extension/docs/InstallNightlyBuild.md references `release` and you've not changed it on this PR (I only see the Tekton folder's markdown doc being changed)